stardis_env3.yml

```diff
@@ -0,0 +1,78 @@
+name: stardis
```

Comment:
We will want to use a conda lock file for this, it makes the environment reproducible.

Reply:
Unable to create lock file for stardis.

Reply:
I think, we put this down as an issue.